Vauhtio/vauhtio-com
# Email behind button

## Description

This PR changes email links behind button press to prevent spam emails. Also change hero cta button to scroll to contact section rather than straight email link

## How to test?

Go to home page and rekry page and verify that hero button scroll user to contact section. Also check that email link is not visible until you press `Näytä sähköposti`

![image](https://user-images.githubusercontent.com/12229968/96370783-b15e8f00-1167-11eb-99e6-0630c19caa70.png)
